TwoNote/app/src/main/java/com/microsoft/device/display/samples/twonote/NoteSelectionListener.kt

```diff
@@ -1,15 +1,40 @@
 package com.microsoft.device.display.samples.twonote
 
+import android.content.Context
+import android.util.Log
 import android.view.ActionMode
 import android.view.Menu
 import android.view.MenuItem
 import android.widget.AbsListView
+import android.widget.ArrayAdapter
 import android.widget.ListView
+import androidx.core.view.get
+import androidx.core.view.iterator
+import androidx.core.view.size
+import com.microsoft.device.display.samples.twonote.model.INode
+import java.io.File
 
-class NoteSelectionListener(private var host: NoteListFragment, private var listView: ListView) : AbsListView.MultiChoiceModeListener {
+class NoteSelectionListener(private var host: NoteListFragment, private var listView: ListView,
+                            private var context: Context, private var arrayAdapter: ArrayAdapter<INode>?) : AbsListView.MultiChoiceModeListener {
```

Comment:
I think you could get the "context" from the host parameter itself? like host.context or host.activity.context I believe

Also I'm going to go look into the code more but maybe there is an easier way to delete where we directly connect to the model instead of using the arrayadapter/listview